notebook_utils/berdl_notebook_utils/spark/database.py

```diff
@@ -189,10 +198,9 @@ def list_tables(spark: SparkSession, namespace: str | None = None) -> list:
             tables_df = spark.sql("SHOW TABLES")
 
         # Extract table names from the DataFrame
-        tables = [row["tableName"] for row in tables_df.collect()]
-        return tables
```

Comment:
no need to set intermediate variable and return it -- just return the function output directly